src/info/langs/language.rs

```diff
@@ -71,17 +76,20 @@ impl std::fmt::Display for LanguagesInfo {
             DynColors::Ansi(AnsiColors::Cyan),
         ];
 
-        let languages: Vec<(String, f64, DynColors)> = prepare_languages(self, &color_palette);
+        let languages: Vec<LanguageDisplayData> = prepare_languages(self, &color_palette);
 
         let mut languages_info = build_language_bar(&languages);
 
-        for (i, (language, perc, circle_color)) in languages.iter().enumerate() {
-            let formatted_number = format!("{:.*}", 1, perc);
-            let circle = "\u{25CF}".color(*circle_color);
+        for (i, language_display_data) in languages.iter().enumerate() {
+            let formatted_number = format!("{:.*}", 1, language_display_data.percentage);
+            let circle = language_display_data
+                .icon
+                .color(language_display_data.chip_color);
             let language_str = format!(
                 "{} {} ",
                 circle,
-                format!("{language} ({formatted_number} %)").color(self.info_color)
+                format!("{0} ({formatted_number} %)", language_display_data.language)
+                    .color(self.info_color)
```

Comment:
should `circle` be renamed to chip also?

Reply:
yes